js/views/userPageVw.js

```diff
@@ -1413,6 +1437,92 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator');
+
+    $targ.addClass('loading');
+
+    var user = this.model.get('user');
+    var self = this;
+
+    var modList = {};
+    modList.moderators = user.moderators;
+    modList.moderators[modList.moderators.length] = this.userProfile.get('profile').guid;
+    user.moderators = modList.moderators;
```

Comment:
This can be optimized. Using user as a reference to the user object in the model mostly just makes things more complicated, and creates the potential for bugs, since it may not be clear changing user changes user in the model. 

I suggest just building the modList array like this:

modList.moderators = this.model.get('user').moderators;
modList.moderators.push(this.pageID);